### PR TITLE
Runbook nitpicks

### PR DESCRIPTION
**Pull request by encukou**

### CODE REVIEW

File: operations/runbooks.rst [hunk 1]
```diff
@@ -89,6 +89,4 @@ Zápis z jednorázového elektronického hlasování
 
       .. image:: ../_static/voting/YYYY-MM-DDTHH-MM-SS.png
 
-      .. image:: ../_static/voting/YYYY-MM-DDTHH-MM-SS.png
```
Comment (honzajavorek) [inline operations/runbooks.rst]: This was supposed to be an example of when multiple screenshots are needed, but I agree it looks like a typo 😀

Comment (encukou) [reply operations/runbooks.rst]: I think that:
* if you just need one screenshot, this definitely looks like a typo
* if you need several screenshots, you can figure that out even without the duplicated line

Comment (encukou) [reply operations/runbooks.rst]: But I can revert if you want :)

Comment (honzajavorek) [reply operations/runbooks.rst]: No, I approved it. No strong opinions, I agree with the change. It was more of an explanation of my original intentions for context.

**Comment by encukou** [comment]
Circle CI says:
```
Warning, treated as error:
/home/circleci/repo/guides/promotion.rst.rst:17:broken link: https://ep2019.europython.eu/ (HTTPSConnectionPool(host='ep2019.europython.eu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:852)'),)))
Exited with code 2
```
The link works for me, so I'll merge.

**Comment by honzajavorek** [comment]
Yeah, can be temporary. That’s why I moved it to a separate CI step, which
isn’t required to pass for merging.

**Pull request merged**